qemu/tests/virtio_serial_hotplug_max_chardevs.py

```diff
@@ -107,8 +107,8 @@ def run_bg_test():
     try:
         char_devices[0].hotplug(vm.monitor, vm.devices.qemu_version)
     except QMPCmdError as e:
-        if "attempt to add duplicate property '%s'" % char_devices[0] \
-                not in str(e.data):
+        if not ("duplicate property '%s'" % char_devices[0] in str(e.data)
+                or "'%s' already exists" % char_devices[0] in str(e.data)):
```

Comment:
Hello @zhenyzha, is it(`"attempt to add duplicate property '%s'"` --> `"duplicate property '%s'"`) sure that it won't have any effect?

Reply:
In fact, that's a piece of negative test information, as long as it triggered an exception and correctly displays the reason, I think it has no effect.

Reply:
OK, thanks.